docs/_articles/en/codebuilder/cb-diffs.md

```diff
@@ -2,23 +2,19 @@
 title: What's Different
 lang: en
 ---
+
 ## What’s Different in Code Builder
 
 Code Builder unleashes the power of Salesforce Extensions for VS Code and offers the same rich features as the desktop version. No matter whether you're using VS Code on your desktop or Code Builder from a browser, the Salesforce extensions you access are the same. While the extensions generally function the same way regardless of how they're accessed, there are some nuances to be aware of:
 
-* Code Builder is installed as a managed package, and it comes with everything you need: VS Code, Salesforce extensions, and Salesforce CLI. No installation required. 
```

Comment:
@sbudhirajadoc - I suggest changing the last sentence to "No installation required on your local machined."  The first sentence talks about installing the managed package and then we say "no installs" so I feel like we need to clarify that a little bit more.